src/realm-settings/LocalizationTab.tsx

```diff
@@ -364,37 +553,38 @@ export const LocalizationTab = ({
                   </Select>
                 </ToolbarItem>
               }
-              toolbarItem={
-                <Button
-                  data-testid="add-bundle-button"
-                  isDisabled={!formState.isSubmitSuccessful}
-                  onClick={() => setAddMessageBundleModalOpen(true)}
-                >
-                  {t("addMessageBundle")}
-                </Button>
-              }
-              searchPlaceholderKey=" "
-              emptyState={
+            >
+              {messageBundles.length === 0 && !filter && (
                 <ListEmptyState
                   hasIcon={true}
                   message={t("noMessageBundles")}
                   instructions={t("noMessageBundlesInstructions")}
                   onPrimaryAction={handleModalToggle}
-                  primaryActionText={t("addMessageBundle")}
                 />
-              }
-              canSelectAll
-              columns={[
-                {
-                  name: "Key",
-                  cellRenderer: (row) => row[0],
-                },
-                {
-                  name: "Value",
-                  cellRenderer: (row) => row[1],
-                },
-              ]}
-            />
+              )}
+              {messageBundles.length === 0 && filter && (
+                <ListEmptyState
+                  hasIcon={true}
+                  icon={SearchIcon}
+                  isSearchVariant={true}
```

Comment:
Boolean props that are `true` can have their value omitted.
```suggestion
                  isSearchVariant
```